src/devices/nm-device.c

```diff
@@ -4227,7 +4227,7 @@ device_init_static_sriov_num_vfs (NMDevice *self)
 		num_vfs = _nm_utils_ascii_str_to_int64 (value, 10, 0, G_MAXINT32, -1);
 		if (num_vfs >= 0) {
 			nm_platform_link_set_sriov_params (nm_device_get_platform (self),
-			                                   priv->ifindex, num_vfs, -1);
+			                                   priv->ifindex, num_vfs, NM_TERNARY_DEFAULT);
```

Comment:
in the commit message, there is a "Fixes" comment, but it's not clear what the problem with the previous implementation is? Is there an actual bug? Then the commit message should highlight it (I didn't see it).

If it's only a code improvement (of which I am in favour), then the Fixes comment seems a bit too strong. Generally, I really like Fixes if they actually fix a problem. But here?